src/goTest.ts

```diff
@@ -122,7 +122,8 @@ function goTest(config: TestConfig): Thenable<boolean> {
 		outputChannel.show(2);
 		let buildFlags: string[] = vscode.workspace.getConfiguration('go')['buildFlags'];
 		let buildTags: string = vscode.workspace.getConfiguration('go')['buildTags'];
-		let args = ['test', '-v', '-timeout', config.timeout, '-tags', buildTags, ...buildFlags];
+		let testFlags: string[] = vscode.workspace.getConfiguration('go')['testFlags'];
+		let args = ['test', ...testFlags, '-timeout', config.timeout, '-tags', buildTags, ...buildFlags];
```

Comment:
I like the idea of using `testFlags` for build/test flags to the test command. Should we remove support for `buildFlags` here, given there is no distinction in `go test` between build and test flags? Having both is redundant and confusing to me given your new option.